package.json

```diff
@@ -74,6 +74,7 @@
     "urijs": "^1.19.11"
   },
   "jest": {
+    "maxWorkers": 5,
```

Comment:
@cmars I've been noticing that jest tests occasionally fail with `Call retries were exceeded - ChildProcessWorker.initialize (node_modules/jest-worker/build/workers/ChildProcessWorker.js:193:21)` - [failure here](https://app.circleci.com/pipelines/github/snyk/sweater-comb/683/workflows/1c318698-c280-4249-98b1-b77ad5667253/jobs/782)

Think this is due to: https://github.com/facebook/jest/issues/8769#issuecomment-919811307 - in which case this should limit the parallel workers running in ci